message_test.go

```diff
@@ -182,6 +182,34 @@ func TestParseMessageError(t *testing.T) {
 	}
 }
 
+func TestMarshalMessageChecksum(t *testing.T) {
+	source := net.ParseIP("2001:db8::10")
+	destination := net.ParseIP("2001:db8::1")
+	mac, _ := net.ParseMAC("86:31:82:05:ce:9a")
+	message := &ndp.NeighborAdvertisement{
+		Solicited:     true,
+		Override:      true,
+		TargetAddress: source,
+		Options: []ndp.Option{&ndp.LinkLayerAddress{
+			Direction: ndp.Target,
+			Addr:      mac,
+		}},
+	}
+	bytes, err := ndp.MarshalMessageChecksum(message, source, destination)
+	if err != nil {
+		t.Fatal("Unexpected error")
```

Comment:
Add error contents in the message `t.Fatalf("failed to marshal message: %v", err)`.